OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -115,10 +114,12 @@ public class UpdatePath
 				new RefactorResourceLevelAnimating(),
 			}),
 
-			new UpdatePath("playtest-20190106", new UpdateRule[]
+			new UpdatePath("playtest-20190106", "playtest-20190209", new UpdateRule[]
 			{
 				new RemoveAttackSuicides(),
-			})
+			}),
+
+			new UpdatePath("playtest-20190209", new UpdateRule[0])
```

Comment:
(17:45:52) antares79: and do we want an empty update path for playtest-20190302?
(17:46:25) pchote: yeah, otherwise the utility will error out when people want to update mods based on it
(17:46:29) pchote: even if it is a no-op

Reply:
Done.